### PR TITLE
Express開発前準備

### PR DESCRIPTION
**Pull request by nirot1r**

### CODE REVIEW

File: api.config.js [hunk 1]
```diff
@@ -0,0 +1,3 @@
+export default {
```
Comment (TinyKitten) [inline api.config.js]: Please consider using .env

File: package.json [hunk 2]
```diff
@@ -4,13 +4,21 @@
...
-    "jest": "^24.5.0"
+    "@babel/cli": "^7.2.3",
+    "@babel/core": "^7.3.4",
+    "@babel/node": "^7.2.2",
+    "@babel/preset-env": "^7.3.4",
+    "jest": "^24.5.0",
+    "supertest": "^4.0.2"
   },
   "scripts": {
+    "start": "babel-node src/index",
```
Comment (TinyKitten) [inline package.json]: Do not use babel-node

Comment (TinyKitten) [reply package.json]: Run after build instead.

File: api.config.js [hunk 3]
```diff
@@ -0,0 +1,3 @@
...
+  port: 3030
```
Comment (TinyKitten) [inline api.config.js]: Why don't you use port 1919?

File: api.config.js (hunk 1 above)
Comment (nirot1r) [reply api.config.js]: We will use `*.config.js`

File: package.json (hunk 2 above)
Comment (TinyKitten) [reply package.json]: We should using babel-node only in development environment.

File: package.json [hunk 4]
```diff
@@ -4,13 +4,21 @@
...
-    "jest": "^24.5.0"
+    "@babel/cli": "^7.2.3",
+    "@babel/core": "^7.3.4",
+    "@babel/node": "^7.2.2",
+    "@babel/preset-env": "^7.3.4",
+    "jest": "^24.5.0",
+    "supertest": "^4.0.2"
   },
   "scripts": {
+    "dev": "babel-node src/index",
```
Comment (TinyKitten) [inline package.json]: Please add `start` and `build` npm scripts.

Comment (nirot1r) [reply package.json]: done.

**Comment by TinyKitten** [comment]
issagen

**Pull request merged**